pkg/database/postgres/index.go

```diff
@@ -0,0 +1,35 @@
+package postgres
+
+import (
+	"fmt"
+	"strings"
+
+	schemasv1alpha2 "github.com/schemahero/schemahero/pkg/apis/schemas/v1alpha2"
+	"github.com/schemahero/schemahero/pkg/database/types"
+)
+
+func RemoveIndexStatement(tableName string, index *types.Index) string {
+	return fmt.Sprintf("drop index %s", index.Name)
```

Comment:
A space (or some other special character) in the name would break this.  How about double quotes around the name (replace `%s` with `%q`)?